libs/ledger-live-common/src/exchange/swap/hooks/live-app-migration/useIsCurrencySupported.ts

```diff
@@ -0,0 +1,27 @@
+import { CryptoOrTokenCurrency } from "@ledgerhq/types-cryptoassets";
+import { isCryptoCurrency } from "../../../../currencies";
+
+type UseIsCurrencySupportedProps = {
+  currencyFrom?: CryptoOrTokenCurrency;
+  params: {
+    families: string;
+    currencies: Array<unknown>;
+  };
+};
+
+export function useIsCurrencySupported({ currencyFrom, params }: UseIsCurrencySupportedProps) {
+  const { families, currencies } = params || {};
+
+  if (!currencyFrom || (!families && !currencies)) {
+    return false;
+  }
```

Comment:
I do not think you need this "if"

If "currencyFrom" swapWebApp button will be disabled but it is good go to swap-web-app
If enabled is true without more info, means all the currencies are working fine.

```suggestion
if (!currencyFrom || (!families && !currencies)) {
    return true;
  }
```

Old code: 
```
if (!currencyFrom || (!families && !currencies)) {
    return enabled;
  }